integration-tests/container-image/maven-invoker-way/src/it/container-native-main/pom.xml

```diff
@@ -40,7 +40,7 @@
       <plugin>
         <groupId>io.quarkus</groupId>
         <artifactId>quarkus-maven-plugin</artifactId>
-        <version>@project.version@</version>
+        <version>999-SNAPSHOT</version>
```

Comment:
Is there a reason why you did that, IIRC, it can be annoying with releases.

Reply:
And same for the other occurrence.

Reply:
good catch! It was failing and I needed a way to run it (without having to run all the tests)

Reply:
Fixed.